corehq/motech/repeaters/migrations/0016_add_indexes.py

```diff
@@ -12,49 +12,61 @@ class Migration(migrations.Migration):
         migrations.SeparateDatabaseAndState(
             state_operations=[
                 migrations.AlterField(
-                    model_name="repeatrecord",
-                    name="state",
-                    field=models.PositiveSmallIntegerField(
-                        choices=[
-                            (1, "Pending"),
-                            (2, "Failed"),
-                            (4, "Succeeded"),
-                            (8, "Cancelled"),
-                            (16, "Empty"),
-                            (32, "Invalid Payload"),
-                        ],
-                        db_index=True,
-                        default=1,
-                    ),
+                    model_name="repeater",
+                    name="is_deleted",
+                    field=models.BooleanField(default=False),
                 ),
                 migrations.AddIndex(
                     model_name="repeater",
                     index=models.Index(
-                        condition=models.Q(("is_deleted", False), ("is_paused", False)),
-                        fields=["next_attempt_at"],
-                        name="next_attempt_at_partial_idx",
+                        condition=models.Q(("is_deleted", False)),
+                        fields=["id"],
+                        name="is_deleted_partial_idx",
+                    ),
+                ),
+                migrations.AddIndex(
+                    model_name="repeatrecord",
+                    index=models.Index(
+                        condition=models.Q(("state__in", (1, 2))),
```

Comment:
I wonder what are the computational differences in Postgres for `state IN (1, 2)` vs `state = 1 OR state = 2`? I hope they are virtually the same, but it would be a shame if `IN (...)` does something complicated with a loop.